low-level/message_handlers/service_msg_handler.py

```diff
@@ -266,9 +268,8 @@ def _process_msg(self, jsonMsg):
                                             prev substate: {prev_substate}")
 
             # Create a service watchdog message and send it out
-            jsonMsg = ServiceWatchdogMsg(service_name, state, prev_state,
-                            substate, prev_substate, pid, prev_pid).getJson()
-            self._write_internal_msgQ("RabbitMQegressProcessor", jsonMsg)
+            # jsonMsg = ServiceWatchdogMsg(service_name, state, prev_state, substate, prev_substate, pid, prev_pid).getJson()

```

Comment:
We are removing service msg alerts from SystemdWatchdog, I have not removed SystemdWatchdog sensor yet since it handles drives alerts as well. so removed the whole if condition handling service msg alerts from  systemdWatchdog.